src/experimental-bots/spotFiller/spotFillerMultithreaded.ts

```diff
@@ -182,10 +185,7 @@ export class SpotFillerMultithreaded {
 	private slotSubscriber: SlotSubscriber;
 	private globalConfig: GlobalConfig;
 	private seenFillableOrders = new Set<string>();
-	private seenTriggerableOrders = new Set<string>();
```

Comment:
you probably didnt mean to remove this i think

Reply:
it wasnt used